web/packages/teleterm/build_resources/linux/after-remove.tpl

```diff
@@ -31,7 +31,11 @@ BIN=/usr/local/bin
 TSH_SYMLINK_TARGET=$BIN/tsh
 
 # Remove the link to the Electron app binary.
-rm -f "$BIN/${executable}"
+if type update-alternatives >/dev/null 2>&1; then
+  update-alternatives --remove "${executable}" "$BIN/${executable}"
```

Comment:
It appears that the call to `update-alternatives` in after-remove appears to be incorrect in electron-builder's template.

On Fedora, installation works without problems, but on remove I got this:

```
/usr/local/bin/teleport-connect has not been configured as an alternative for teleport-connect
warning: %postun(teleport-connect-1.0.0_dev-1.aarch64) scriptlet failed, exit status 2
```

I tried to execute the commands manually and I got some other error, then it occured to me that the call might be wrong.

```
$ update-alternatives --help
alternatives version 1.24 - Copyright (C) 2001 Red Hat, Inc.
This may be freely redistributed under the terms of the GNU Public License.

usage: alternatives --install <link> <name> <path> <priority>
                    [--initscript <service>]
                    [--family <family>]
                    [--follower <follower_link> <follower_name> <follower_path>]*
       alternatives --remove <name> <path>
```

```suggestion
  update-alternatives --remove "${executable}" "$APP/${executable}"
```

(but you also need to copy `APP` from after-install and define it in after-remove)

After applying this change, the error on Fedora is gone. The warning on Ubuntu still persists though.

Reply:
I should have tested it on Fedora 😞 My only excuse is that I was slightly tired of testing the app before opening the PR.

Anyway, good catch! WDYT about opening a PR with the fix in electron-builder?

Reply:
I mean if you want to then go for it. 😏